packages/truffle/cli.webpack.config.js

```diff
@@ -66,6 +67,11 @@ module.exports = {
       "BUNDLE_CHAIN_FILENAME": JSON.stringify("chain.bundled.js")
     }),
 
+    new webpack.DefinePlugin({
+      "BUNDLE_VERSION": JSON.stringify(pkg.version),
+      "BUNDLE_ANALYTICS_FILENAME": JSON.stringify("analytics.bundled.js")
+    }),
```

Comment:
ah I think this should be merged into the one above... we can get rid of the extra `BUNDLE_VERSION` and just include `BUNDLE_ANALYTICS_FILENAME` in the same object